src/modules/users/components/input-fields/confirm-password-input.tsx

```diff
@@ -0,0 +1,19 @@
+import { useRef } from 'react';
+import Box from '@mui/material/Box';
+import { BasicInput } from '#/modules/users/components';
+import { type FormInputProps } from '../../types';
+import { isInputFieldEmpty } from '#/utils';
+import { inputStyles as styles } from '../../styles/input.styles';
```

Comment:
Use mapped paths. Here and in other files too.

Reply:
fixed